CONTRIBUTING.md

```diff
@@ -44,38 +56,45 @@ Scan through our [existing issues](https://github.com/dfe-analytical-services/an
 ### Make Changes
 
 Key things to remember when making or proposing changes:
-- Where guidance already exists elsewhere it should be linked to rather than duplicated
-- When adding images, save them in the /images folder
-- When adding files for download, save them in the /resources folder
-- Don't commit rendered .html files for pages
-- If your changes remove or edit a pre-existing anchor link, consider how that will be redirected for users who may have bookmarked it
+
+-   Where guidance already exists elsewhere it should be linked to rather than duplicated
+
+-   When adding images, save them in the /images folder
+
+-   When adding files for download, save them in the /resources folder - Don't commit rendered .html files for pages
+
+-   If your changes remove or edit a pre-existing anchor link, consider how that will be redirected for users who may have bookmarked it
 
 #### Make changes in the UI
 
 Click **Edit this page** at the bottom of the right hand table of contents of any page to make small changes such as a typo, sentence fix, or a broken link. This takes you to the `.Qmd` file where you can make your changes and [create a pull request](#pull-request) for a review.
 
 #### Make changes locally
 
-1. Clone or fork the repository.
+1.  Clone or fork the repository.
 
-2. Open the repository in your editor of choice, e.g. R Studio.
+2.  Open the repository in your editor of choice, e.g. R Studio.
 
-3. Create a working branch and start with your changes!
+3.  Create a working branch and start with your changes!
 
-4. Commit and push the changes to your working branch.
+4.  Commit and push the changes to your working branch.
 
-### Pull Request
+### Pull Request {#pull-request}
```

Comment:
Do we know why this is tagged in this way specifically and other headers aren't?